addons/account/models/account_move_line.py

```diff
@@ -226,6 +226,8 @@ class AccountMoveLine(models.Model):
         tracking=True,
         help="Tags assigned to this line by the tax creating it, if any. It determines its impact on financial reports.",
     )
+    extra_tax_data = fields.Json()
```

Comment:
maybe add a string and help to the field?

Reply:
@antoine162 I'm not sure I can find a clear helper enough meaningfull for a dev indicating the purpose of this field :sweat_smile: "Technical field holding custom data for the taxes computation engine, touche pas à ca petit con" maybe? :thinking:

Reply:
Yeah, just the last bit can be reworded slightly but otherwise sounds good :trollface: